llpc/context/llpcCompiler.cpp

```diff
@@ -1675,6 +1737,8 @@ Result Compiler::BuildComputePipeline(
 {
     BinaryData elfBin = {};
 
+    bool buildingRelocatableElf = CanUseRelocatableComputeShaderElf(&pPipelineInfo->cs);
+    
```

Comment:
Prefer to check this directly in the if-statement below, rather than adding a new local variable. (After you removed the duplication due to preprocessor #if).